README.rst

```diff
@@ -44,25 +55,76 @@ Install pytorch via your preferred way.
    pip install git+git://github.com/lgvaz/mantisshrimp.git
    pip install -U 'git+https://github.com/cocodataset/cocoapi.git#subdirectory=PythonAPI'
 
-Quick start
------------
+Quick Example: How to train the **Wheat Dataset**
```

Comment:
Should we link to the wheat dataset kaggle page?

Or even better, we can use Pennfundan, I added functions for automatically downloading the dataset

```
from mantisshrimp.hub.pennfundan import *
source = get_pennfundan_data()
parser = PennFundanParser(source)
```

The problem is that the `PennFundanParser` is quite complicated.

Reply:
Maybe a toy dataset with COCO would be even better. The goal is to let the users to quick play with it in Colab.

Reply:
I'll take a look at that balloon dataset I've been talking about, if I remember correctly it was COCO format

Reply:
Yes, let's stay far away from Facial Recognition!